Resources/docs/navbar_messages.md

```diff
@@ -76,6 +76,12 @@ class MessageSubscriber implements EventSubscriberInterface
         $userModel->setName($myUser->getUsername());
         $message = new MessageModel($userModel, 'Hello world');
         $event->addMessage($message);
+        
+        /*
+         * You can also set the total number of messages wich could be different from those displayed in the navbar
```

Comment:
Minor typo: `You can also set the total number of messages which could be different from those displayed in the navbar.`